MantidQt/CustomInterfaces/test/ReflSettingsTabPresenterTest.h

```diff
@@ -144,6 +144,9 @@ class ReflSettingsTabPresenterTest : public CxxTest::TestSuite {
     EXPECT_CALL(mockView, getProcessingInstructions())
         .Times(Exactly(1))
         .WillOnce(Return("\"3,4\""));
+    EXPECT_CALL(mockView, getTransmissionRuns())
+        .Times(Exactly(1))
+        .WillOnce(Return("INTER00013463.nxs,INTER00013464.nxs"));
```

Comment:
I suggest just using "INTER00013463, INTER00013464" here, or, even better, just "13463, 13464". The reason is that this way the test will reflect what will happen in reality, as users will only enter run numbers, not the full name and extension of the transmission workspace.